domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -81,6 +82,18 @@ public async Task SeedData(IEnumerable<TransferAgreement> transferAgreements)
         }
     }
 
+    public async Task SeedConnections(IEnumerable<Connection> connections)
```

Comment:
The same method for TransferAgreements is named `SeedData`. There should probably be some renaming of `SeedData`

Reply:
Renamed